AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/reference.md

```diff
@@ -0,0 +1,11 @@
+# Window reference
+
+This section provides details on the properties for the following windows or components:
+
+- [Alembic Exporter component](ref_Exporter.html), which allows you to export GameObjects, Cameras, and Points (in the Particle System).
+- [Alembic Point Cloud component](ref_PointCloud.html), which allows you to customize the shape and behavior of the point cloud.
+- [Alembic Point Renderer component](ref_PointRenderer.html), which allows you to customize the appearance of the particles in the point cloud.
+- [Alembic Recorder Clip component](ref_Recorder.html), which allows you to record animation from the Alembic file.
```

Comment:
I think it should be "record animation **to** an Alembic file"

Reply:
Good catch, thanks!